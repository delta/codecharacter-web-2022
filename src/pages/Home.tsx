export default function Home(): JSX.Element {
  return (
    <main>
      <h1>Home</h1>
    </main>
  );
}
