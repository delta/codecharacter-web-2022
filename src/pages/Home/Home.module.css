.mainContainer {
  margin-left: -60px;
  background-color: white;
}

.jumbotron {
  width: 100%;
  height: calc(100vh - 60px);

  display: flex;
  justify-content: center;
  align-items: center;

  /* background: url('../../assets/jumbotron.jpeg'); */
  background: url('https://images.unsplash.com/photo-1542831371-29b0f74f9713?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8cHJvZ3JhbW1pbmd8ZW58MHx8MHx8&w=1000&q=80');
  background-size: cover;
}

.jumbotron > div {
  font-size: 64px;
  color: white;
  font-weight: bold;
}

.delta {
  display: flex;
  background-color: #eee;
}

.delta > div {
  margin: auto;
}

.heart {
  color: red;
}
