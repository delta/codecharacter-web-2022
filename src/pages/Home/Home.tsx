import Footer from '../../components/Home/Footer/Footer';
import Glitchtext from '../../components/Home/Glitchtext/Glitchtext';
import Rows from '../../components/Home/Row/Rows';
import styles from './Home.module.css';

export default function Home(): JSX.Element {
  return (
    <main className={styles.mainContainer}>
      <div className={styles.jumbotron}>
        <Glitchtext value="Code Character" />
      </div>
      <Rows />
      <Footer />
      <div className={styles.delta}>
        <div>
          Made with <span className={styles.heart}>‚ù§</span> by{' '}
          <a href="https://delta.nitt.edu/">Delta Force</a>
        </div>
      </div>
    </main>
  );
}
