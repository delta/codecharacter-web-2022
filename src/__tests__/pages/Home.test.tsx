import * as renderer from 'react-test-renderer';
import Home from '../../pages/Home';

it('renders correctly', () => {
  const tree = renderer.create(<Home></Home>).toJSON();
  expect(tree).toMatchSnapshot();
});
