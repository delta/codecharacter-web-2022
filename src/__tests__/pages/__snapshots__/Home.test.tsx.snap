// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main>
  <h1>
    Home
  </h1>
</main>
`;
